Conceptpower+Spring/src/test/java/edu/asu/conceptpower/servlet/rest/ConceptIDLookupTest.java

```diff
@@ -0,0 +1,116 @@
+        String conceptId = "CONf375adff-dde7-4536-9e62-f80328f800d0";
+        String wordNetIds = "W-ID1, W-ID2";
+        String expectedResponseString = "<conceptpowerReply xmlns:digitalHPS=\"http://www.digitalhps.org/\">"
+                + "<digitalHPS:conceptEntry><digitalHPS:id concept_id=\"" + conceptId + "\" "
+                + "concept_uri=\"null\">null</digitalHPS:id><digitalHPS:lemma>null</digitalHPS:lemma>"
+                + "<digitalHPS:pos>null</digitalHPS:pos><digitalHPS:description>null</digitalHPS:description>"
+                + "<digitalHPS:conceptList>null</digitalHPS:conceptList><digitalHPS:creator_id>"
+                + "</digitalHPS:creator_id><digitalHPS:equal_to></digitalHPS:equal_to>"
+                + "<digitalHPS:modified_by></digitalHPS:modified_by><digitalHPS:similar_to>"
+                + "</digitalHPS:similar_to><digitalHPS:synonym_ids></digitalHPS:synonym_ids><digitalHPS:type ></digitalHPS:type>"
+                + "<digitalHPS:deleted>false</digitalHPS:deleted><digitalHPS:wordnet_id>" + wordNetIds
+                + "</digitalHPS:wordnet_id>"
+                + "</digitalHPS:conceptEntry></conceptpowerReply>";
+        ConceptEntry entry = new ConceptEntry();
+        entry.setId(conceptId);
+        entry.setWordnetId(wordNetIds);
+        Mockito.when(dictManager.getConceptEntry(conceptId)).thenReturn(entry);
+        ResponseEntity<String> response = conceptIDLookup
+                .getConceptById("http://www.digitalhps.org/concepts/" + conceptId);
+        Assert.assertEquals(HttpStatus.OK, response.getStatusCode());
+        Assert.assertEquals(expectedResponseString, response.getBody());
+
+    }
+
+    @Test
+    public void testGetConcepyByIdWithGenericIDs() throws Exception {
+        String wordNetIds = "W-ID01, W-ID02";
+        String conceptId = "CONf375adff-dde7-4536-9e62-f80328f800d0";
+        String expectedResponse = "<conceptpowerReply xmlns:digitalHPS=\"http://www.digitalhps.org/\"><digitalHPS:conceptEntry>"
+                + "<digitalHPS:id concept_id=\"" + conceptId + "\" concept_uri=\"null\">null</digitalHPS:id>"
+                + "<digitalHPS:lemma>pony</digitalHPS:lemma><digitalHPS:pos>NOUN</digitalHPS:pos>"
+                + "<digitalHPS:description>Description</digitalHPS:description>"
+                + "<digitalHPS:conceptList>null</digitalHPS:conceptList>"
+                + "<digitalHPS:creator_id></digitalHPS:creator_id><digitalHPS:equal_to>equals</digitalHPS:equal_to>"
+                + "<digitalHPS:modified_by></digitalHPS:modified_by><digitalHPS:similar_to>similar</digitalHPS:similar_to>"
+                + "<digitalHPS:synonym_ids>SYN-01</digitalHPS:synonym_ids><digitalHPS:type ></digitalHPS:type>"
+                + "<digitalHPS:deleted>false</digitalHPS:deleted>"
+                + "<digitalHPS:wordnet_id>" + wordNetIds
+                + "</digitalHPS:wordnet_id></digitalHPS:conceptEntry></conceptpowerReply>";
```

Comment:
Let's put that into a file instead of hardcode it here. Will be easier to maintain. Just put it right next to the test case and use getResourceAsStream or something like that.